<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TreeATE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTreeWidget" name="treeWidget_Units">
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Path</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Result</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Standard</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Start Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Spend Time</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>6</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>6</height>
       </size>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Users</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuProjects">
    <property name="title">
     <string>&amp;Projects</string>
    </property>
    <addaction name="actionLoading"/>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
    <addaction name="actionProperty"/>
    <addaction name="separator"/>
    <addaction name="action_Edit"/>
    <addaction name="actionOption"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_Sys">
    <property name="title">
     <string>&amp;System</string>
    </property>
    <addaction name="action_Results"/>
    <addaction name="separator"/>
    <addaction name="actionSys_options"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="actionHelp_for_project"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionShrink_items"/>
    <addaction name="actionSpread_items"/>
    <addaction name="separator"/>
    <addaction name="actionWindows"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProjects"/>
   <addaction name="menu_Sys"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoading"/>
   <addaction name="actionPlay"/>
   <addaction name="actionStop"/>
   <addaction name="actionProperty"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_Property">
   <property name="windowTitle">
    <string>Property</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tableWidget_Property">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Description</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_propertyName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_SaveCfg">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/modify.png</normaloff>:/modify.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Log">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>68</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_log">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTextBrowser" name="textBrowser_Log"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Rst">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>33</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Status</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_rst">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_rst">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionLoading">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/loading.png</normaloff>:/loading.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Loading...</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/play.png</normaloff>:/play.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Play (F5)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pause.png</normaloff>:/pause.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop (F8)</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionProperty">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/modify.png</normaloff>:/modify.png</iconset>
   </property>
   <property name="text">
    <string>Proper&amp;ty...</string>
   </property>
   <property name="toolTip">
    <string>Property</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>&amp;Login...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>&amp;Help...</string>
   </property>
  </action>
  <action name="actionHelp_for_project">
   <property name="text">
    <string>Help for &amp;project...</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>&amp;Log</string>
   </property>
  </action>
  <action name="actionOutput">
   <property name="text">
    <string>&amp;Output</string>
   </property>
  </action>
  <action name="actionSpread_items">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/add.png</normaloff>:/add.png</iconset>
   </property>
   <property name="text">
    <string>Spread items (Ctrl+=)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionShrink_items">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/minus.png</normaloff>:/minus.png</iconset>
   </property>
   <property name="text">
    <string>Shrink items (Ctrl+-)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionSys_options">
   <property name="text">
    <string>System o&amp;ptions...</string>
   </property>
  </action>
  <action name="actionOption">
   <property name="text">
    <string>&amp;Option...</string>
   </property>
  </action>
  <action name="action_Results">
   <property name="text">
    <string>&amp;Results...</string>
   </property>
  </action>
  <action name="actionWindows">
   <property name="text">
    <string>Windows</string>
   </property>
  </action>
  <action name="action_Edit">
   <property name="text">
    <string>&amp;Devlop...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>treeWidget_Units</tabstop>
  <tabstop>textBrowser_Log</tabstop>
  <tabstop>tableWidget_Property</tabstop>
  <tabstop>pushButton_SaveCfg</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
